CHANGELOG.md

```diff
@@ -1,3 +1,237 @@
+## v0.6.0 - 2020-12-31
+
+We're happy to announce Lokomotive v0.6.0 (Flying Scotsman).
+
+This release includes several new features, many component updates, and a new platform - [Tinkerbell](https://tinkerbell.org/).
+
+### Changes in v0.6.0
+
+- Remove deprecated CoreOS mentions from AWS ([#1245](https://github.com/kinvolk/lokomotive/pull/1245)) and bare metal ([#1246](https://github.com/kinvolk/lokomotive/pull/1246)).
+
+#### Kubernetes updates
+
+- Update Kubernetes to v1.19.4 and AKS to v1.18.10 ([#1189](https://github.com/kinvolk/lokomotive/pull/1189)).
+
+#### Component updates
+
+- Update External DNS to v0.7.4 ([#1115](https://github.com/kinvolk/lokomotive/pull/1115)).
```

Comment:
Shouldn't the component names be as shown in `lokoctl components list` 
example:  `external-dns`